[32mINFO    [0m juju.model:__init__.py:2301 Deploying ch:amd64/focal/mattermost-k8s-27
[32mINFO    [0m juju.model:__init__.py:3254 Waiting for model:
  mattermost-k8s/0 [allocating] waiting: installing agent
[32mINFO    [0m juju.model:__init__.py:3254 Waiting for model:
  mattermost-k8s/0 [idle] waiting: Waiting for database relation
[33mWARNING [0m juju.model:__init__.py:1683 relate is deprecated and will be removed. Use integrate instead.
[32mINFO    [0m juju.model:__init__.py:3254 Waiting for model:
  mattermost-k8s/0 [idle] waiting: Waiting for database relation
[32mINFO    [0m juju.model:__init__.py:3254 Waiting for model:
  mattermost-k8s/0 [executing] active: 
[32mINFO    [0m juju.model:__init__.py:3254 Waiting for model:
  mattermost-k8s/0 [idle] active:
[32mINFO    [0m pytest_operator.plugin:plugin.py:936 Model status:

Model    Controller          Cloud/Region        Version  SLA          Timestamp
testing  concierge-microk8s  microk8s/localhost  3.6.2    unsupported  02:12:17Z

App                       Version  Status  Scale  Charm                     Channel        Rev  Address         Exposed  Message
pgbouncer-k8s             1.21.0   active      2  pgbouncer-k8s                              0  10.152.183.114  no       
postgresql-k8s            14.15    active      3  postgresql-k8s            14/edge        507  10.152.183.240  no       
postgresql-test-app                active      1  postgresql-test-app       latest/edge    360  10.152.183.227  no       received database credentials of the first database
self-signed-certificates           active      1  self-signed-certificates  latest/stable  155  10.152.183.212  no       

Unit                         Workload  Agent      Address       Ports  Message
pgbouncer-k8s/0*             active    executing  10.1.226.137         
pgbouncer-k8s/1              active    executing  10.1.226.138         
postgresql-k8s/0*            active    idle       10.1.226.144         Primary
postgresql-k8s/1             active    idle       10.1.226.145         
postgresql-k8s/2             active    idle       10.1.226.146         
postgresql-test-app/0*       active    idle       10.1.226.139         received database credentials of the first database
self-signed-certificates/0*  active    idle       10.1.226.143         

[32mINFO    [0m pytest_operator.plugin:plugin.py:942 Juju error logs:

unit-postgresql-k8s-2: 02:00:07 ERROR unit.postgresql-k8s/2.juju-log certificates:9: Cannot push TLS certificates: <ops.pebble.ConnectionError ("Could not connect to Pebble: socket not found at '/charm/containers/postgresql/pebble.socket' (container restarted?)",)>
unit-postgresql-k8s-0: 02:00:08 ERROR unit.postgresql-k8s/0.juju-log certificates:9: Cannot push TLS certificates: <ops.pebble.ConnectionError ("Could not connect to Pebble: socket not found at '/charm/containers/postgresql/pebble.socket' (container restarted?)",)>
unit-postgresql-k8s-2: 02:00:10 ERROR unit.postgresql-k8s/2.juju-log Cannot push TLS certificates: <ops.pebble.ConnectionError ("Could not connect to Pebble: socket not found at '/charm/containers/postgresql/pebble.socket' (container restarted?)",)>
unit-postgresql-k8s-0: 02:00:10 ERROR unit.postgresql-k8s/0.juju-log Cannot push TLS certificates: <ops.pebble.ConnectionError ("Could not connect to Pebble: socket not found at '/charm/containers/postgresql/pebble.socket' (container restarted?)",)>
unit-postgresql-k8s-2: 02:00:11 ERROR unit.postgresql-k8s/2.juju-log database:7: Cannot push TLS certificates: <ops.pebble.ConnectionError ("Could not connect to Pebble: socket not found at '/charm/containers/postgresql/pebble.socket' (container restarted?)",)>
unit-postgresql-k8s-1: 02:00:11 ERROR unit.postgresql-k8s/1.juju-log certificates:9: Cannot push TLS certificates: <ops.pebble.ConnectionError ("Could not connect to Pebble: socket not found at '/charm/containers/postgresql/pebble.socket' (container restarted?)",)>
unit-postgresql-k8s-2: 02:00:12 ERROR unit.postgresql-k8s/2.juju-log Cannot push TLS certificates: <ops.pebble.ConnectionError ("Could not connect to Pebble: socket not found at '/charm/containers/postgresql/pebble.socket' (container restarted?)",)>
unit-postgresql-k8s-1: 02:00:13 ERROR unit.postgresql-k8s/1.juju-log Cannot push TLS certificates: <ops.pebble.ConnectionError ("Could not connect to Pebble: socket not found at '/charm/containers/postgresql/pebble.socket' (container restarted?)",)>
unit-postgresql-k8s-2: 02:00:14 ERROR unit.postgresql-k8s/2.juju-log certificates:9: Cannot push TLS certificates: <ops.pebble.ConnectionError ("Could not connect to Pebble: socket not found at '/charm/containers/postgresql/pebble.socket' (container restarted?)",)>
unit-postgresql-k8s-1: 02:00:14 ERROR unit.postgresql-k8s/1.juju-log database:7: Cannot push TLS certificates: <ops.pebble.ConnectionError ("Could not connect to Pebble: socket not found at '/charm/containers/postgresql/pebble.socket' (container restarted?)",)>
unit-postgresql-k8s-2: 02:00:16 ERROR unit.postgresql-k8s/2.juju-log database:7: Cannot push TLS certificates: <ops.pebble.ConnectionError ("Could not connect to Pebble: socket not found at '/charm/containers/postgresql/pebble.socket' (container restarted?)",)>
unit-postgresql-k8s-1: 02:00:16 ERROR unit.postgresql-k8s/1.juju-log Cannot push TLS certificates: <ops.pebble.ConnectionError ("Could not connect to Pebble: socket not found at '/charm/containers/postgresql/pebble.socket' (container restarted?)",)>
unit-postgresql-k8s-1: 02:00:17 ERROR unit.postgresql-k8s/1.juju-log database:7: Cannot push TLS certificates: <ops.pebble.ConnectionError ("Could not connect to Pebble: socket not found at '/charm/containers/postgresql/pebble.socket' (container restarted?)",)>
unit-postgresql-k8s-2: 02:00:17 ERROR unit.postgresql-k8s/2.juju-log database:7: Cannot push TLS certificates: <ops.pebble.ConnectionError ("Could not connect to Pebble: socket not found at '/charm/containers/postgresql/pebble.socket' (container restarted?)",)>
unit-postgresql-k8s-2: 02:00:19 ERROR unit.postgresql-k8s/2.juju-log database:7: Cannot push TLS certificates: <ops.pebble.ConnectionError ("Could not connect to Pebble: socket not found at '/charm/containers/postgresql/pebble.socket' (container restarted?)",)>
unit-postgresql-k8s-1: 02:00:19 ERROR unit.postgresql-k8s/1.juju-log database:7: Cannot push TLS certificates: <ops.pebble.ConnectionError ("Could not connect to Pebble: socket not found at '/charm/containers/postgresql/pebble.socket' (container restarted?)",)>
unit-postgresql-k8s-1: 02:00:20 ERROR unit.postgresql-k8s/1.juju-log database-peers:6: Cannot push TLS certificates: <ops.pebble.ConnectionError ("Could not connect to Pebble: socket not found at '/charm/containers/postgresql/pebble.socket' (container restarted?)",)>
unit-postgresql-k8s-2: 02:00:20 ERROR unit.postgresql-k8s/2.juju-log restart:4: Cannot push TLS certificates: <ops.pebble.ConnectionError ("Could not connect to Pebble: socket not found at '/charm/containers/postgresql/pebble.socket' (container restarted?)",)>
unit-postgresql-k8s-2: 02:00:21 ERROR unit.postgresql-k8s/2.juju-log upgrade:5: Cannot push TLS certificates: <ops.pebble.ConnectionError ("Could not connect to Pebble: socket not found at '/charm/containers/postgresql/pebble.socket' (container restarted?)",)>
unit-postgresql-k8s-1: 02:00:21 ERROR unit.postgresql-k8s/1.juju-log upgrade:5: Cannot push TLS certificates: <ops.pebble.ConnectionError ("Could not connect to Pebble: socket not found at '/charm/containers/postgresql/pebble.socket' (container restarted?)",)>
unit-postgresql-k8s-0: 02:00:22 ERROR unit.postgresql-k8s/0.juju-log Cannot push TLS certificates: <ops.pebble.ConnectionError ("Could not connect to Pebble: socket not found at '/charm/containers/postgresql/pebble.socket' (container restarted?)",)>
unit-postgresql-k8s-2: 02:00:33 ERROR unit.postgresql-k8s/2.juju-log database-peers:6: Cannot push TLS certificates: <ops.pebble.ConnectionError ("Could not connect to Pebble: socket not found at '/charm/containers/postgresql/pebble.socket' (container restarted?)",)>
unit-postgresql-k8s-1: 02:00:33 ERROR unit.postgresql-k8s/1.juju-log database-peers:6: Cannot push TLS certificates: <ops.pebble.ConnectionError ("Could not connect to Pebble: socket not found at '/charm/containers/postgresql/pebble.socket' (container restarted?)",)>
unit-postgresql-k8s-0: 02:00:34 ERROR unit.postgresql-k8s/0.juju-log certificates:9: Cannot push TLS certificates: <ops.pebble.ConnectionError ("Could not connect to Pebble: socket not found at '/charm/containers/postgresql/pebble.socket' (container restarted?)",)>
unit-postgresql-k8s-1: 02:00:35 ERROR unit.postgresql-k8s/1.juju-log database-peers:6: Cannot push TLS certificates: <ops.pebble.ConnectionError ("Could not connect to Pebble: socket not found at '/charm/containers/postgresql/pebble.socket' (container restarted?)",)>
unit-postgresql-k8s-2: 02:00:35 ERROR unit.postgresql-k8s/2.juju-log restart:4: Cannot push TLS certificates: <ops.pebble.ConnectionError ("Could not connect to Pebble: socket not found at '/charm/containers/postgresql/pebble.socket' (container restarted?)",)>
unit-postgresql-k8s-1: 02:00:36 ERROR unit.postgresql-k8s/1.juju-log database-peers:6: Cannot push TLS certificates: <ops.pebble.ConnectionError ("Could not connect to Pebble: socket not found at '/charm/containers/postgresql/pebble.socket' (container restarted?)",)>
unit-postgresql-k8s-2: 02:00:36 ERROR unit.postgresql-k8s/2.juju-log upgrade:5: Cannot push TLS certificates: <ops.pebble.ConnectionError ("Could not connect to Pebble: socket not found at '/charm/containers/postgresql/pebble.socket' (container restarted?)",)>
unit-postgresql-k8s-1: 02:00:37 ERROR unit.postgresql-k8s/1.juju-log database-peers:6: Cannot push TLS certificates: <ops.pebble.ConnectionError ("Could not connect to Pebble: socket not found at '/charm/containers/postgresql/pebble.socket' (container restarted?)",)>
unit-postgresql-k8s-2: 02:00:38 ERROR unit.postgresql-k8s/2.juju-log database-peers:6: Cannot push TLS certificates: <ops.pebble.ConnectionError ("Could not connect to Pebble: socket not found at '/charm/containers/postgresql/pebble.socket' (container restarted?)",)>
unit-postgresql-k8s-2: 02:00:39 ERROR unit.postgresql-k8s/2.juju-log restart:4: Cannot push TLS certificates: <ops.pebble.ConnectionError ("Could not connect to Pebble: socket not found at '/charm/containers/postgresql/pebble.socket' (container restarted?)",)>
unit-postgresql-k8s-1: 02:00:39 ERROR unit.postgresql-k8s/1.juju-log upgrade:5: Cannot push TLS certificates: <ops.pebble.ConnectionError ("Could not connect to Pebble: socket not found at '/charm/containers/postgresql/pebble.socket' (container restarted?)",)>
unit-postgresql-k8s-2: 02:00:41 ERROR unit.postgresql-k8s/2.juju-log database-peers:6: Cannot push TLS certificates: <ops.pebble.ConnectionError ("Could not connect to Pebble: socket not found at '/charm/containers/postgresql/pebble.socket' (container restarted?)",)>
unit-postgresql-k8s-1: 02:00:41 ERROR unit.postgresql-k8s/1.juju-log restart:4: Cannot push TLS certificates: <ops.pebble.ConnectionError ("Could not connect to Pebble: socket not found at '/charm/containers/postgresql/pebble.socket' (container restarted?)",)>
unit-postgresql-k8s-2: 02:00:42 ERROR unit.postgresql-k8s/2.juju-log database-peers:6: Cannot push TLS certificates: <ops.pebble.ConnectionError ("Could not connect to Pebble: socket not found at '/charm/containers/postgresql/pebble.socket' (container restarted?)",)>
unit-postgresql-k8s-1: 02:00:42 ERROR unit.postgresql-k8s/1.juju-log upgrade:5: Cannot push TLS certificates: <ops.pebble.ConnectionError ("Could not connect to Pebble: socket not found at '/charm/containers/postgresql/pebble.socket' (container restarted?)",)>
unit-postgresql-k8s-2: 02:00:43 ERROR unit.postgresql-k8s/2.juju-log database-peers:6: Cannot push TLS certificates: <ops.pebble.ConnectionError ("Could not connect to Pebble: socket not found at '/charm/containers/postgresql/pebble.socket' (container restarted?)",)>
unit-postgresql-k8s-2: 02:00:44 ERROR unit.postgresql-k8s/2.juju-log restart:4: Cannot push TLS certificates: <ops.pebble.ConnectionError ("Could not connect to Pebble: socket not found at '/charm/containers/postgresql/pebble.socket' (container restarted?)",)>
unit-postgresql-k8s-0: 02:00:46 ERROR unit.postgresql-k8s/0.juju-log database:7: Cannot push TLS certificates: <ops.pebble.ConnectionError ("Could not connect to Pebble: socket not found at '/charm/containers/postgresql/pebble.socket' (container restarted?)",)>
unit-postgresql-k8s-2: 02:00:46 ERROR unit.postgresql-k8s/2.juju-log upgrade:5: Cannot push TLS certificates: <ops.pebble.ConnectionError ("Could not connect to Pebble: socket not found at '/charm/containers/postgresql/pebble.socket' (container restarted?)",)>
unit-postgresql-k8s-1: 02:00:53 ERROR unit.postgresql-k8s/1.juju-log restart:4: Cannot push TLS certificates: <ops.pebble.ConnectionError ("Could not connect to Pebble: socket not found at '/charm/containers/postgresql/pebble.socket' (container restarted?)",)>
unit-postgresql-k8s-1: 02:00:55 ERROR unit.postgresql-k8s/1.juju-log restart:4: Cannot push TLS certificates: <ops.pebble.ConnectionError ("Could not connect to Pebble: socket not found at '/charm/containers/postgresql/pebble.socket' (container restarted?)",)>
unit-postgresql-k8s-1: 02:00:56 ERROR unit.postgresql-k8s/1.juju-log restart:4: Cannot push TLS certificates: <ops.pebble.ConnectionError ("Could not connect to Pebble: socket not found at '/charm/containers/postgresql/pebble.socket' (container restarted?)",)>
unit-postgresql-k8s-1: 02:00:58 ERROR unit.postgresql-k8s/1.juju-log upgrade:5: Cannot push TLS certificates: <ops.pebble.ConnectionError ("Could not connect to Pebble: socket not found at '/charm/containers/postgresql/pebble.socket' (container restarted?)",)>
unit-postgresql-k8s-0: 02:00:59 ERROR unit.postgresql-k8s/0.juju-log database:7: Cannot push TLS certificates: <ops.pebble.ConnectionError ("Could not connect to Pebble: socket not found at '/charm/containers/postgresql/pebble.socket' (container restarted?)",)>
unit-postgresql-k8s-2: 02:00:59 ERROR unit.postgresql-k8s/2.juju-log upgrade:5: Cannot push TLS certificates: <ops.pebble.ConnectionError ("Could not connect to Pebble: socket not found at '/charm/containers/postgresql/pebble.socket' (container restarted?)",)>
unit-postgresql-k8s-1: 02:01:00 ERROR unit.postgresql-k8s/1.juju-log upgrade:5: Cannot push TLS certificates: <ops.pebble.ConnectionError ("Could not connect to Pebble: socket not found at '/charm/containers/postgresql/pebble.socket' (container restarted?)",)>
unit-postgresql-k8s-2: 02:01:00 ERROR unit.postgresql-k8s/2.juju-log upgrade:5: Cannot push TLS certificates: <ops.pebble.ConnectionError ("Could not connect to Pebble: socket not found at '/charm/containers/postgresql/pebble.socket' (container restarted?)",)>
unit-postgresql-k8s-0: 02:01:10 ERROR unit.postgresql-k8s/0.juju-log database:7: Cannot push TLS certificates: <ops.pebble.ConnectionError ("Could not connect to Pebble: socket not found at '/charm/containers/postgresql/pebble.socket' (container restarted?)",)>
unit-postgresql-k8s-1: 02:01:12 ERROR unit.postgresql-k8s/1.juju-log Cannot push TLS certificates: <ops.pebble.ConnectionError ("Could not connect to Pebble: socket not found at '/charm/containers/postgresql/pebble.socket' (container restarted?)",)>
unit-postgresql-k8s-2: 02:01:12 ERROR unit.postgresql-k8s/2.juju-log Cannot push TLS certificates: <ops.pebble.ConnectionError ("Could not connect to Pebble: socket not found at '/charm/containers/postgresql/pebble.socket' (container restarted?)",)>
unit-postgresql-k8s-1: 02:01:16 ERROR unit.postgresql-k8s/1.juju-log Cannot push TLS certificates: <ops.pebble.ConnectionError ("Could not connect to Pebble: socket not found at '/charm/containers/postgresql/pebble.socket' (container restarted?)",)>
unit-postgresql-k8s-2: 02:01:16 ERROR unit.postgresql-k8s/2.juju-log Cannot push TLS certificates: <ops.pebble.ConnectionError ("Could not connect to Pebble: socket not found at '/charm/containers/postgresql/pebble.socket' (container restarted?)",)>
unit-postgresql-k8s-0: 02:01:22 ERROR unit.postgresql-k8s/0.juju-log database:7: Cannot push TLS certificates: <ops.pebble.ConnectionError ("Could not connect to Pebble: socket not found at '/charm/containers/postgresql/pebble.socket' (container restarted?)",)>
unit-postgresql-k8s-1: 02:01:27 ERROR unit.postgresql-k8s/1.juju-log Cannot push TLS certificates: <ops.pebble.ConnectionError ("Could not connect to Pebble: socket not found at '/charm/containers/postgresql/pebble.socket' (container restarted?)",)>
unit-postgresql-k8s-2: 02:01:27 ERROR unit.postgresql-k8s/2.juju-log Cannot push TLS certificates: <ops.pebble.ConnectionError ("Could not connect to Pebble: socket not found at '/charm/containers/postgresql/pebble.socket' (container restarted?)",)>
unit-postgresql-k8s-0: 02:01:34 ERROR unit.postgresql-k8s/0.juju-log database:7: Cannot push TLS certificates: <ops.pebble.ConnectionError ("Could not connect to Pebble: socket not found at '/charm/containers/postgresql/pebble.socket' (container restarted?)",)>
unit-postgresql-k8s-1: 02:01:36 ERROR unit.postgresql-k8s/1.juju-log Cannot push TLS certificates: <ops.pebble.ConnectionError ("Could not connect to Pebble: socket not found at '/charm/containers/postgresql/pebble.socket' (container restarted?)",)>
unit-postgresql-k8s-2: 02:01:37 ERROR unit.postgresql-k8s/2.juju-log Cannot push TLS certificates: <ops.pebble.ConnectionError ("Could not connect to Pebble: socket not found at '/charm/containers/postgresql/pebble.socket' (container restarted?)",)>
unit-postgresql-k8s-0: 02:01:45 ERROR unit.postgresql-k8s/0.juju-log database-peers:6: Cannot push TLS certificates: <ops.pebble.ConnectionError ("Could not connect to Pebble: socket not found at '/charm/containers/postgresql/pebble.socket' (container restarted?)",)>
unit-postgresql-k8s-1: 02:01:48 ERROR unit.postgresql-k8s/1.juju-log Cannot push TLS certificates: <ops.pebble.ConnectionError ("Could not connect to Pebble: socket not found at '/charm/containers/postgresql/pebble.socket' (container restarted?)",)>
unit-postgresql-k8s-2: 02:01:48 ERROR unit.postgresql-k8s/2.juju-log Cannot push TLS certificates: <ops.pebble.ConnectionError ("Could not connect to Pebble: socket not found at '/charm/containers/postgresql/pebble.socket' (container restarted?)",)>
unit-postgresql-k8s-2: 02:02:00 ERROR unit.postgresql-k8s/2.juju-log Cannot push TLS certificates: <ops.pebble.ConnectionError ("Could not connect to Pebble: socket not found at '/charm/containers/postgresql/pebble.socket' (container restarted?)",)>
unit-postgresql-k8s-1: 02:02:00 ERROR unit.postgresql-k8s/1.juju-log Cannot push TLS certificates: <ops.pebble.ConnectionError ("Could not connect to Pebble: socket not found at '/charm/containers/postgresql/pebble.socket' (container restarted?)",)>
unit-postgresql-k8s-2: 02:03:56 ERROR unit.postgresql-k8s/2.juju-log pgdata folder not found in /var/lib/postgresql/data/pgdata

[32mINFO    [0m pytest_operator.plugin:plugin.py:1024 Forgetting model main...